.secrets.baseline

```diff
@@ -162,7 +162,11 @@
         "filename": "apps/bbsync/tests/test_integration.py",
         "hashed_secret": "3c3b274d119ff5a5ec6c1e215c1cb794d9973ac1",
         "is_verified": false,
+<<<<<<< HEAD
         "line_number": 70,
+=======
+        "line_number": 80,
+>>>>>>> 7943700 (keep generating BZ ID Jira tracker label)
```

Comment:
this merge conflict will break the secrets.

Reply:
The final version is OK. I can fix this in the history.